pdfgen.c

```diff
@@ -1708,7 +1720,7 @@ int pdf_get_font_text_width(struct pdf_doc *pdf, const char *font_name,
 static const char *find_word_break(const char *string)
 {
     /* Skip over the actual word */
-    while (string && *string && !isspace(*string))
+    while (string && *string && (*string < 0 || !isspace(*string)))
```

Comment:
What was the issue here that needed `*string < 0`? I'm wondering if that will give issues with weird UTF-8 encodings... Or actually possibly it fixes UTF-8 issues, since we're doing this char-by-char.

Reply:
Yes it was to resolve a UTF-8 issue. When compiled with MSVC passing characters outside of the ascii range (which the leading bytes of a UTF-8 sequence typically are) to this causes a debug assertion printout.